gcp_python_repo_tools/appengine_sdk.py

```diff
@@ -48,11 +52,34 @@ def get_gae_versions():
             continue
 
         versions_and_urls.append(
-            (match.groups(), release['mediaLink']))
+            ([int(x) for x in match.groups()], release['mediaLink']))
 
     return sorted(versions_and_urls, key=lambda x: x[0])
 
 
+def check_existing(destination, latest_version):
```

Comment:
A more descriptive method name might be helpful - something like `is_gae_up_to_date` (though I guess then you'd have to negate the returned inequality)